README.md

```diff
@@ -17,22 +22,22 @@ single gvcf file which can be used by the joint-discovery workflow.
 #### Outputs 
 - One GVCF file and its index
 
-### joint-discovery-gatk :
+### JointGenotyping.wdl :
 This WDL implements the joint calling and VQSR filtering portion of the 
 GATK Best Practices for germline SNP and Indel discovery 
-in human whole-genome sequencing (WGS).
+in human whole-genome sequencing (WGS). The workflow accept a sample map 
+file with 50 or more GVCFs and produces a multisample VCF.
 
 *NOTE:*  
-*- joint-discovery-gatk4-local.wdl is a slightly modified version of the 
-original to support users interested in running the workflow locally.*  
-*- joint-discovery-gatk4-fc.wdl is a slightly modified version of the 
-original to support users interested in running the workflow firecloud with and
-using an array of gvcfs as input.*
+*- JointGenotyping-terra.wdl is a slightly modified version of the 
```

Comment:
You told me when we met that the user has to make the sample map after running the GVCF workflow on all samples, right?  If there's no method/workflow for that, we should point it out.

Reply:
Added a link to [generate-sample-map](https://portal.firecloud.org/?return=terra#methods/gatk/generate-sample-map/1) for users interested in creating a sample map on Terra